packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import { findDOMNode } from 'react-dom';
 import { BaseDecorator } from './BaseDecorator';
-import { getWindow, hoistStatics } from '../../Utilities';
+import { EventGroup, getWindow, css } from '@uifabric/utilities';
```

Comment:
Should be this per convention:
```suggestion
import { EventGroup, getWindow, css } from '../../Utilities';
```
Also we're trying to get rid of EventGroup and use the new EventListener utility instead--see #11949 for example. (edit: I see there's a conversation about that elsewhere, but I agree with not using EventGroup here)

Reply:
I need to backport these changes to Fabric 6.0, so it's best not to get carried away with the refactor and usage of new dependencies.